.rubocop.yml

```diff
@@ -45,6 +45,9 @@ Lint/AmbiguousRegexpLiteral:
 CyclomaticComplexity:
   Max: 15 # should be set to 6, too many for now
 
+ParameterLists:
+  Max: 6
```

Comment:
6 is echt veel te lang normaal. Je kunt het bij de functie uitzetten of we kunnen hier `CountKeywordArgs: false` zetten.